### DISCUSSION TITLE
Table component

### DISCUSSION THREAD

**Discussion by d8vjork** [tailwindlabs/headlessui#399] [Ideas]
Hey there,

Do you guys have something planned around tables? I know they're components in Tailwind UI, but it will be really cool if we have a component here that makes sorting, filtering, paginating (with filters and sorts applied) much easier.

Also keep doing the great work!

**Comment by nasa8x** [comment]
I hope that the Team will prioritize this component. Because the demand is so much. I am also looking forward to it.

**Comment by nkutinha** [comment]
Expressing my interest towards a table component 👀

**Comment by timini** [comment]
mee three

**Comment by phyolim** [comment]
There's already a pretty extensive headless table library if you're using React - react-table

**Reply by timini** [reply]
Yes, thats what i ended up using
